src/robusta/core/sinks/robusta/prometheus_discovery_utils.py

```diff
@@ -39,6 +38,30 @@ def __init__(self, discovery_period_sec: int, global_config: dict):
     def get_status(self) -> PrometheusHealthStatus:
         return self.status
 
+    def get_cluster_avg_cpu(self) -> Optional[float]:
+        cpu_query = os.getenv("OVERRIDE_CLUSTER_CPU_AVG_QUERY",
+                              f'avg_over_time(sum(irate(container_cpu_usage_seconds_total{{}}[5m]))[1h:])')
```

Comment:
I'm not sure this is the correct query
The goal, is to show the avg **node** cpu utilization in the cluster, during the last hour (this metric is for `container` which has a different meaning I think)

Also, why are we using both `[5m]` and `[1h:]`?